src/coreclr/src/jit/lower.cpp

```diff
@@ -3688,9 +3688,18 @@ void Lowering::InsertPInvokeCallProlog(GenTreeCall* call)
         GenTree* frameAddr =
             new (comp, GT_LCL_VAR_ADDR) GenTreeLclVar(GT_LCL_VAR_ADDR, TYP_BYREF, comp->lvaInlinedPInvokeFrameVar);
 
+#ifdef _TARGET_X86_
```

Comment:
```suggestion
#ifdef TARGET_X86
```
The ifdefs were renamed recently.

Reply:
> This is for Windows only

For clarity, you can also move all logic inside `InlinedCallFrame::UpdateRegDisplay` that deals with stack argument sizes under `UNIX_X86_ABI`.

Reply:
Ok, so what's the Unix ABI? Does the caller pop the arguments from the stack?

Reply:
Right